### PR TITLE
LPS-129251 Create Poshi test

### PR DESCRIPTION
**Pull request by brunofariasdeo**
Related Issue: https://issues.liferay.com/browse/LPS-129251

### CODE REVIEW

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase [hunk 1]
```diff
@@ -3928,6 +3928,94 @@ definition {
 			fieldValue = "Checkbox 1");
 	}
 
+	@description = "This is a use case for LPS-125860."
+	@priority = "5"
+	test SubmitFormWithRequiredConfirmationTextField {
+		property test.name.skip.portal.instance = "Forms#SubmitFormWithRequiredConfirmationTextField";
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase:3934]: @brunofariasdeo, please add the following properties:

- property portal.acceptance = "false";
- property portal.upstream = "false";

Comment (brunofariasdeo) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase:3936]: Done!

**Comment by brunofariasdeo** [comment]
ci:test:sf

**Comment by brunofariasdeo** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @brunofariasdeo, can you please check if these failures are related to your changes? Thanks

**Comment by brunofariasdeo** [comment]
ci:forward

**Pull request closed without merging**